sklearn/model_selection/_search.py

```diff
@@ -1066,11 +1152,19 @@ class RandomizedSearchCV(BaseSearchCV):
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validation strategies that can be used here.
 
-    refit : boolean, default=True
+    refit : boolean, or string default=True
```

Comment:
Hmm, you want the default value to be `True` if scoring is single metric and `False` if `multimetric`?

@jnothman fine by you?